src/Features/CSharp/Portable/Completion/CompletionProviders/DeclarationNameCompletionProvider.cs

```diff
@@ -219,9 +219,15 @@ private Glyph GetGlyph(SymbolKind kind, Accessibility declaredAccessibility)
             var namingStyleOptions = options.GetOption(SimplificationOptions.NamingPreferences);
             var rules = namingStyleOptions.CreateRules().NamingRules.Concat(s_BuiltInRules);
             var result = new Dictionary<string, SymbolKind>();
-            foreach (var symbolKind in declarationInfo.PossibleSymbolKinds)
+            foreach (var kind in declarationInfo.PossibleSymbolKinds)
             {
-                var kind = new SymbolKindOrTypeKind(symbolKind);
+                // There's no special glyph for local functions.
```

Comment:
There probably should be. The Glyph enum has everything - ClassPublic, StructPublic, FieldPublic, FieldPrivate, Local, Parameter... local functions are clearly missing.

Reply:
Can you open a bug?

Reply:
filed #26327